#! /bin/sh /usr/share/dpatch/dpatch-run
## 55_strip_logarchivepath.dpatch by Omni Flux <omniflux@omniflux.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: stripping the log_archive_path value from cgi.cfg (#578232) 
@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' pkg-nagios3~/cgi/cgiutils.c pkg-nagios3/cgi/cgiutils.c
--- pkg-nagios3~/cgi/cgiutils.c	2012-05-07 20:43:10.000000000 +0200
+++ pkg-nagios3/cgi/cgiutils.c	2012-05-07 22:00:06.155062976 +0200
@@ -500,7 +500,7 @@
 			temp_buffer = strtok(NULL, "\n");
 			strncpy(log_archive_path, (temp_buffer == NULL) ? "" : temp_buffer, sizeof(log_archive_path));
 			log_archive_path[sizeof(log_archive_path) - 1] = '\x0';
-			strip(physical_html_path);
+			strip(log_archive_path);
 			if(log_archive_path[strlen(log_archive_path) - 1] != '/' && (strlen(log_archive_path) < sizeof(log_archive_path) - 1))
 				strcat(log_archive_path, "/");
 			}
